pkg/azuredisk/azure_common_windows.go

```diff
@@ -51,8 +52,11 @@ func scsiHostRescan(io ioHandler, m *mount.SafeFormatAndMount) {
 }
 
 // search Windows disk number by LUN
-func findDiskByLun(lun int, iohandler ioHandler, m *mount.SafeFormatAndMount) (string, error) {
-	if proxy, ok := m.Interface.(mounter.CSIProxyMounter); ok {
+func findDiskByLun(lun int, iohandler azureutils.IOHandler, m *mount.SafeFormatAndMount) (string, error) {
+	if proxy, ok := m.Interface.(*mounter.CSIProxyMounter); ok {
```

Comment:
`CSIProxyMounter` is now an interface in the Mandalay repo. Remove the `*` here and it should work. Also, you'll need to remove any statements that contain a type assertion for `CSIProxyMounterV1Beta` as they are unnecessary. (I should really just make this change to the upstream repo to avoid this merge issue.)

Reply:
Fixed